src/schema/routers/schemaRouter.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import { FactoryFunction } from 'tsyringe';
+import { SchemaController } from '../controllers/schemaController';
+
+const schemaRouterFactory: FactoryFunction<Router> = (dependencyContainer) => {
+  const router = Router();
+  const controller = dependencyContainer.resolve(SchemaController);
+
+  router.get('/', controller.getSchemas.bind(controller));
```

Comment:
`controller.getSchemas` is an arrow function, `bind` is redundant

Reply:
Done